.rubocop_todo.yml

```diff
@@ -0,0 +1,40 @@
+# This configuration was generated by
```

Comment:
Actually... why are we introducing this file?

Reply:
it's an autogenerated file, which:
- creates a todo list of things you should work on (it was MUCH larger before I started working on the rubocop fixes)
- makes the build/travis pass

It's a typical and official strategy: make rubocop passing and create a todo list of offences that in the perfect world should be fixed at some point.
The rubocop update brought some changes and it was out of scope of this task to rebuild most of the already existing specs (still, I fixed 70% of them)

Reply:
I dunno... but it sounds like a hack. Rubocop is failing and we create an ignore list just to have green build. Right now version on master is passing so changes that fail the build are introduced in this PR tree here - so my point is i think that instead of 70% we need to fix 100% 😉 
or don't upgrade rubocop in this PR.
My guts tell me that nobody will ever fix this todo list.

Reply:
yes, that's the first thought but at least:

- you know what you should work on (this is mostly style/syntax so it's not critical)
- you can separate past from the presence, focus on new stuff and make it passing 100% (not as like on sg, when you touch old files you ignore all the PR bot notes - is better? I don't think so)
- it's better to update rubocop due to security alerts than not doing it because of work due to rubocop offences, even with no security alerts for current version of rubocop it's good to be up to date and simply ignore some default configuration changes introducing offences than stay with the old version because we don't want to fix some new cops etc.
- finally, there are no stable set of u2i/nielsen rules that we should ignore (exclusions in the basic rubocop configuration) - there's a chance there's no need to fix 100% if we make the rules a bit more relaxed (e.g. if we follow http://relaxed.ruby.style)

Reply:
Sounds like a lot of excuses to me 😜
The problem is - we do not disable default configuration. We just make a todo list that we commit to repository.

Reply:
Verions of `rubocop-rspec` got updated, rubocop started whining about things that were fine previously. If you update gems on `master` you'll get the same result and builds will start failing. Issues it's complaining about were not introduced by this PR, they have been in the code for a long time. 
I don't think it's crucial to sort this out at once and now, since it can be a lot of effort, but it's good to know that there is something to do to fix these issues systematically.

Reply:
Ok I totally get it :) .... I just feel that the "correct way" to do this is create a separate PR - upgrade rubocop, fix all the issues. 
What I feel happened here was - upgrade rubocop, fix 70% of the issues, create a todo list...

Reply:
@nowNick if it's just an excuse, feel free to just do it! ;) and to the same on SG

Reply:
Ok :) I'll do it.